derrida/books/tests/test_views.py

```diff
@@ -120,9 +120,17 @@ def test_instance_list_view(self):
             extant_bks.filter(is_annotated=True).count()
         # multiple facets should return both
         response = self.client.get(list_view_url, {'pub_place': ['Paris', 'Pfullingen']})
-        # fixture has 12 published in Paris and 1 in Pfulligen
+        # fixture has 13 published in Paris and 1 in Pfulligen
         assert len(response.context['object_list']) == 14
 
+        # multiple facets should also return counts that are joined by OR within
+        # the same facet, if any of these are 0, it means the facets
+        # within the same field used AND logic
+        # all pub place counts are at least 1 and come as tuples of (name, count)
+        pub_place = response.context['facets']['fields']['pub_place']
+        for item in pub_place:
+            assert item[1] != 0
```

Comment:
Could be written in a single line as `assert not any(item[1] == 0 for item in pub_place)`